images-list

```diff
@@ -3,4 +3,13 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+grafana/grafana rancher/grafana-grafana 7.1.5
+istio/kubectl rancher/istio-kubectl 1.5.10
 library/busybox rancher/library-busybox 1.31.1
+library/nginx rancher/library-nginx 1.19.2-alpine
+openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.1.0
+prom/alertmanager rancher/prom-alertmanager v0.21.0
+prom/node-exporter rancher/prom-node-exporter v1.0.1
+prom/prometheus rancher/prom-prometheus v2.18.2
+thanosio/thanos rancher/thanosio-thanos v0.15.0
+us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.7.3
```

Comment:
@PennyScissors  Do you think we can add in v2 rancher-monitoring images into this PR as well? It's gonna need the following additional images:
```
jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.3.0
jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.2.1
squareup/ghostunnel rancher/squareup-ghostunnel v1.5.2
kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
directxman12/k8s-prometheus-adapter-amd64 rancher/directxman12-k8s-prometheus-adapter-amd64 v0.6.0
bats/bats rancher/bats-bats v1.1.0
curlimages/curl rancher/curlimages-curl 7.70.0
busybox rancher/busybox 1.31.1
```

I can create it as a separate PR, but since it will be referring to a common subset of images (i.e. grafana, alertmanager, node-exporter, prometheus), it seems like it would be easier to keep them in one PR.

Reply:
Sure, I'll add them

Reply:
@aiyengar2 Just noticed this image `busybox rancher/busybox 1.31.1` is already in images-list as `library/busybox rancher/library-busybox 1.31.1`. You might want to update the chart that uses it.

Reply:
Good call out, we do use it. Tracked in https://github.com/rancher/rancher/issues/28978